src/tezos_rpc/http.ml

```diff
@@ -58,9 +58,9 @@ let http_post_data_encoding ~node_uri ~path ~of_yojson ~data =
    resumed when `feed` is called, `feed` is then driven by an string Lwt_stream.
 *)
 let make_lazy_lexbuf read =
-  let open EffectHandlers in
+  let open Effect in
   let module M = struct
-    type _ eff += Poll_data : unit eff
+    type _ Effect.t += Poll_data : unit Effect.t
```

Comment:
btw where can I find documentation for the Effect stuff?

Reply:
Not sure, I read about it here https://github.com/ocaml/ocaml/blob/trunk/stdlib/effect.mli